can-param.js

```diff
@@ -1,6 +1,8 @@
 "use strict";
 var namespace = require("can-namespace");
 
+var standardsMode = false;
```

Comment:
Does this mean that we need to make sure there is only one version of `can-param` in the app? Now that this module is stateful, it could cause unexpected behavior if the version where you `setStandardsMode` is not the version used by another part of the app.

Reply:
After some discussion, I added a multiple version guard.

Reply:
I don't think we need the multiple version guard.  I would prefer this is not a singleton.